### PR TITLE
SCP-2025 - Even out the layout of the metadata form

### PR DESCRIPTION
**Pull request by palas**
This PR applies the CSS grid layout to the form for editing metadata, so that fields and buttons are aligned.

Deployed to: https://pablo.marlowe.iohkdev.io/

Pre-submit checklist:
- Branch
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested
- If you updated any cabal files or added Haskell packages:
    - [x] `nix-shell shell.nix --run updateMaterialized` to update the materialized Nix files
    - [x] Update `hie-*.yaml` files if needed
- If you changed any Haskell files:
    - [x] `nix-shell shell.nix --run fix-stylish-haskell` to fix any formatting issues
- If you changed any Purescript files:
    - [x] `nix-shell shell.nix --run fix-purty` to fix any formatting issues

Pre-merge checklist:
- [x] Someone approved it
- [x] Commits have useful messages
- [x] Review clarifications made it into the code
- [x] History is moderately tidy; or going to squash-merge

### CODE REVIEW

**Review by merivale — APPROVED** [review]
LGTM.

File: marlowe-playground-client/static/css/panels.scss [hunk 1]
```diff
@@ -263,29 +262,37 @@ button.minus-btn:hover {
...
+
+.metadata-form {
+  display: grid;
+  grid-template-columns: auto 1fr auto auto;
+  gap: 0.5rem 0px;
+  width: 100%;
+  box-sizing: border-box;
+  align-items: center;
+  max-width: 75rem;
+  margin: auto;
```
Comment (hrajchert) [inline marlowe-playground-client/static/css/panels.scss:286]: I'd remove `margin: auto`. I don't think we want to align the contents in the center of the panel.

Screen Shot 2021-03-15 at 11 02 57

Screen Shot 2021-03-15 at 11 03 15

File: marlowe-playground-client/static/css/panels.scss [hunk 2]
```diff
@@ -263,29 +262,37 @@ button.minus-btn:hover {
...
 .metadata-not-used {
-  margin-left: 1rem;
-}
\ No newline at end of file
+  row-gap: 1rem;
+}
+
+.metadata-form {
+  display: grid;
+  grid-template-columns: auto 1fr auto auto;
```
Comment (hrajchert) [inline marlowe-playground-client/static/css/panels.scss:280]: Not sure what is the 4th grid column. I gather than first is the label, second is the input, third is the button, and the 4th?

**Review by hrajchert — CHANGES_REQUESTED** [review]
Small change in the alignment and then good to go

File: marlowe-playground-client/static/css/panels.scss (hunk 1 above)
Comment (palas) [reply marlowe-playground-client/static/css/panels.scss:286]: Maybe I should make it 100% width, but I think aligning it left doesn't look very good either :/

File: marlowe-playground-client/static/css/panels.scss (hunk 2 above)
Comment (palas) [reply marlowe-playground-client/static/css/panels.scss:280]: That one is for the warning when you have a field that is not in the contract

File: marlowe-playground-client/static/css/panels.scss (hunk 1 above)
Comment (palas) [reply marlowe-playground-client/static/css/panels.scss:286]: Screenshot_20210315_142757

File: marlowe-playground-client/static/css/panels.scss (hunk 2 above)
Comment (palas) [reply marlowe-playground-client/static/css/panels.scss:280]: Screenshot_20210315_142757

File: marlowe-playground-client/static/css/panels.scss (hunk 1 above)
Comment (palas) [reply marlowe-playground-client/static/css/panels.scss:286]: or max-width 80% 🤔

**Pull request merged**